### DISCUSSION TITLE
建议初始化时强制要求设置管理密码并保存在本地文件，弃用生成随机管理密码

### DISCUSSION THREAD

**Discussion by cxkjntm233** [AlistGo/alist#487] [Ideas]
v2.0.9初始化时使用了随机生成管理密码，生成的随机密码我根本不知道在哪里，config.json中没有，在data.db中？？？我还得费力去找生成的随机密码是什么，还不如初始运行时强制要求设置管理密码省事。

**Comment by xhofe** [comment]
第一次运行不管是脚本还是手动都会输出初始密码 且使用命令行
```
./alist -password
```
也能查看，这很费力吗。

**Reply by cxkjntm233** [reply]
> 

谢谢大佬，ssh命令行里面手动出来了，我为了省事用了宝塔的supervisor管理器，它不显示运行日志导致我找不到密码。
